src/odemis/driver/ueye.py

```diff
@@ -695,9 +695,19 @@ def __init__(self, name, role, device=None, **kwargs):
             # Set the format
             # TODO: color mode based on the maximum BPP
             # cf is_DeviceFeature( IS_DEVICE_FEATURE_CMD_GET_SUPPORTED_SENSOR_BIT_DEPTHS)
```

Comment:
Please update that comment. The `TODO` can go, and indicate that we tried GET_SUPPORTED_SENSOR_BIT_DEPTHS, but it's actually not supported on any camera we've tried.